### PR TITLE
Updated go client and lambda function to pull pubkey from okta

### PR DESCRIPTION
**Pull request by stephaniesun88**
- On the CA side, updated LAMBDA function to fetch pubkey from Okta
- On the client side, add okta user in LAMBDA event data

@argh128 @jwalker-ch Please review :)

### CODE REVIEW

**Comment by jwalker-ch** [comment]
The only thing I see is using the variable okta_password within the changes when its actually the API key (okta_api_token) that will be encrypted in KMS. Other than that LGTM

File: bless_client/bless_client.go [hunk 1]
```diff
@@ -57,7 +54,7 @@ func main() {
         log.Fatal("Error creating cert.")
     }
 
-    cert_err := ioutil.WriteFile(certificate_filename, resp.Payload[1:len(resp.Payload) - 3], 0600)
+    cert_err := ioutil.WriteFile(certificate_filename, resp.Payload[17:len(resp.Payload) - 2], 0600)
```
Comment (argh128) [inline bless_client/bless_client.go]: @stephaniesun88  Can you provide some context around these "magic numbers"? stripping first 17 chars?

**Comment by argh128** [comment]
I have one inline question around magic numbers; but it looks great to me.

File: bless_client/bless_client.go (hunk 1 above)
Comment (stephaniesun88) [inline bless_client/bless_client.go]: resp.Payload is a map, looks like
`{
  "certificate": "<SSH cert>"
}`
I found it's a byte array in golang, so I decided to treat it as a string, and get cert as a substring.

**Comment by stephaniesun88** [comment]
@jwalker-ch Renamed to `encrypted_okta_api_token`

**Comment by jwalker-ch** [comment]
Changes lgtm good to merge

**Pull request merged**